src/py/flwr/server/strategy/aggregate.py

```diff
@@ -132,3 +133,18 @@ def _compute_distances(weights: List[NDArrays]) -> NDArray:
             norm = np.linalg.norm(delta)  # type: ignore
             distance_matrix[i, j] = norm**2
     return distance_matrix
+
+
+def aggregate_trimmed_avg(
+    results: List[Tuple[NDArrays, int]], proportiontocut: float
+) -> NDArrays:
+    """Compute trimmed average."""
+    # Create a list of weights and ignore the number of examples
+    weights = [weights for weights, _ in results]
+
+    trimmed_w: NDArrays = [
+        stats.trim_mean(np.asarray(layer), axis=0, proportiontocut=proportiontocut)
```

Comment:
Maybe it would be better to not import an entire external library to do a simple operation like trimmed mean